### ISSUE TITLE
train siamese.py line 107-112

### ISSUE THREAD

**Issue by lucas2606-rs** [ShuGuoJ/3DAES#4]
what s the function of line 107-112? why should 100 random selected samples as test set rather than like you mentioned in paper "the remaining labeled samples as test set"?

**Comment by ShuGuoJ** [comment]
In training the siamese network, we select randomly 100 samples for saving time, which does not have any influence. And in the final fine-tuning phase, 10 samples for every class are used to train the classifier and others serve as testing samples.

**Comment by lucas2606-rs** [comment]
OK, i got it, thx for reply
